lib/matplotlib/tests/test_dates.py

```diff
@@ -243,11 +243,11 @@ def _create_auto_date_locator(date1, date2):
                  '1990-09-01 00:00:00+00:00', '1990-10-01 00:00:00+00:00',
                  '1990-11-01 00:00:00+00:00', '1990-12-01 00:00:00+00:00']
                 ],
-               [datetime.timedelta(days=140),
-                ['1990-01-06 00:00:00+00:00', '1990-01-27 00:00:00+00:00',
-                 '1990-02-17 00:00:00+00:00', '1990-03-10 00:00:00+00:00',
-                 '1990-03-31 00:00:00+00:00', '1990-04-21 00:00:00+00:00',
-                 '1990-05-12 00:00:00+00:00']
+               [datetime.timedelta(days=141),
```

Comment:
I remember talking about this before, but not the conclusion.  Why is it OK that the tests were changed here?